src/PresentationalComponents/ZeroState/ZeroStateFooter.js

```diff
@@ -0,0 +1,112 @@
+import { Title } from '@patternfly/react-core';
+import '../ZeroState/_zero-state.scss';
+
+import {
+    Flex,
+    FlexItem
+} from '@patternfly/react-core/dist/esm/layouts/Flex/index';
+import {
+    Grid,
+    GridItem
+} from '@patternfly/react-core/dist/esm/layouts/Grid/index';
+import React from 'react';
+import zeroStateConstants from './zeroStateConstants';
+import propTypes from 'prop-types';
+
+const ZeroStateFooter = ({
+    appName = 'Vulnerability',
```

Comment:
We'll need to remove this from the code and just set it as `appName` or some sort of default value that isn't a specific app.

Reply:
For what reason? Having Vulnerability as a default shouldn't be an issue. This just lets us have a default. Making more text to use as a default serves to benefit or real purpose. Or maybe im just not seeing the benefits but I dont think this is necessary

Reply:
I just mean, let's just have `appName` then. It doesn't need to equal anything.